### DISCUSSION TITLE
Diference between Mainnet vs Cyan Mainnet

### DISCUSSION THREAD

**Discussion by JuanP0411** [Web3Auth/Web3Auth#660] [Help]
Besides the estimated monthly users, are there any other differences between them?

**Comment by nerzhul250** [comment]
I also have the same doubt, can anyone answer?

**Reply by camilotobar** [reply]
Same :/

**Comment by YZhenY** [comment]
So two main differences between Cyan and Mainnet:
- Cyan can handle more NEW users, (40 new users vs 5 on Mainnet)
- Cyan cluster is based out of US vs Mainnet in Singapore. This affects login speeds for your users

We're migrating to an infrastructure setup where you wont have to choose between the two in the future (Early Nov). Feel free to choose either setups at this point, migration will happen automatically on the new infra where it will be geo-distributed and horizontally scalable to your needs.
